sections/semantics-document-metadata.include

```diff
@@ -405,21 +405,8 @@
 
   <{link/rel}>'s [=supported tokens=] are the keywords defined in [=HTML link types=] which are
   allowed to be used with <{link}> elements, impact the processing model, and are supported by
-  the user agent. The possible [=supported tokens=] are
-  <{link/alternate}>,
-  <code>dns-prefetch</code>,
-  <{link/icon}>,
-  <{link/next}>,
-  <code data-x="rel-pingback">pingback</code>,
```

Comment:
Remove `pingback` from the` rel` attribute value of `<link>` element?

Reply:
probably should do that as a different issue / PR.  better to track things rather than get them all mixed together.  Happy to have you open an issue about that though :)